### ISSUE TITLE
flox --help tells you what flox does

### ISSUE THREAD

**Issue by ghudgins** [flox/flox#387]
# Problem

flox command help does not have a good explanation on the core workflow--relying on documentation and the https://flox.dev website. We need a one line explanation added to flox CLI.

@jennymahmoudi, @rossturk, @ghudgins to brainstorm. Current front runner:
* flox environments provide isolation where you want it (and not where you don't)

Update with AC once we have a proposal for CLI updates.

**Comment by jennymahmoudi** [comment]
+1 from research session today

**Comment by rossturk** [comment]
I take issue with suggesting that flox environments are isolated - I think that someone who is truly looking for isolation will not find it in flox.

This is not a bad thing. True isolation creates overhead and complexity that MOST workloads don't really need. What they need is separation.

If we were to want to use this pattern, perhaps:
"flox envs provide the workload separation you need without the isolation you don't"
"flox envs provide the discrete environments you need without the isolation you don't"

But I actually like:
"flox: dev environments you can take with you"

**Comment by rossturk** [comment]
> flox is a an application & system-level package manager and virtual environment in one. With flox, you create dev environments you can take with you across the SDLC.

I like this a lot!

Non-blocking, I wonder if "application & system-level" is necessary. I can think of several reasons for including it, but it's a punchier statement without.

**Comment by ghudgins** [comment]
combining multiple ideas 

"flox is a virtual environment and both an application & system-level package manager and in one. With flox, you create dev environments that layer and replace dependencies just where it matters which enables sharing and portability across the full software lifecycle."

**Comment by zmitchell** [comment]
"flox is a virtual environment and package manager all in one. With flox you create development environments that provide isolation where it matters, making them portable across the full software lifecycle. The ability to layer environments, transactionally update them, and share them with your colleagues allows you to focus on getting your work done, not on setting up your tooling."

**Comment by jennymahmoudi** [comment]
yes! ross makes a good point about isolation being a bit of a trap, so removing that. I like the last sentence in @zmitchell's rewrite but I don't think it's necessary here to get the main concepts across. So that leaves us with this revised version to include in help:

"flox is a virtual environment and package manager all in one. With flox you create development environments that layer and replace dependencies just where it matters, making them portable across the full software lifecycle."
